src/main/kotlin/com/terraformation/backend/daily/MonitoringPlotElevationScheduler.kt

```diff
@@ -0,0 +1,83 @@
+package com.terraformation.backend.daily
+
+import com.terraformation.backend.config.TerrawareServerConfig
+import com.terraformation.backend.customer.model.SystemUser
+import com.terraformation.backend.db.SRID
+import com.terraformation.backend.log.perClassLogger
+import com.terraformation.backend.time.ClockAdvancedEvent
+import com.terraformation.backend.tracking.db.PlantingSiteStore
+import com.terraformation.backend.tracking.mapbox.MapboxService
+import com.terraformation.backend.tracking.model.AssignedPlotDetails.Companion.SOUTHWEST
+import jakarta.inject.Inject
+import jakarta.inject.Named
+import org.jobrunr.scheduling.JobScheduler
+import org.jobrunr.scheduling.cron.Cron
+import org.locationtech.jts.geom.GeometryFactory
+import org.locationtech.jts.geom.PrecisionModel
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.event.EventListener
+
+@ConditionalOnProperty(TerrawareServerConfig.DAILY_TASKS_ENABLED_PROPERTY, matchIfMissing = true)
+@Named
+class MonitoringPlotElevationScheduler(
+    private val config: TerrawareServerConfig,
+    private val mapboxService: MapboxService,
+    private val plantingSiteStore: PlantingSiteStore,
+    private val systemUser: SystemUser,
+) {
+  private val log = perClassLogger()
+
+  @Inject
+  fun schedule(scheduler: JobScheduler) {
+    if (config.dailyTasks.enabled) {
+      scheduler.scheduleRecurrently<ObservationScheduler>(
+          javaClass.simpleName, Cron.every15minutes()) {
+            updatePlotElevation()
+          }
+    }
+  }
+
+  @Suppress("MemberVisibilityCanBePrivate") // Called by JobRunr
+  fun updatePlotElevation(limit: Int = 50) {
+    systemUser.run {
+      val plots =
+          try {
+            plantingSiteStore.fetchMonitoringPlotsWithoutElevation(limit)
+          } catch (e: Exception) {
+            log.warn("Failed to fetch monitoring plots without elevation data: ${e.message}")
+            return@run
+          }
+
+      val geometryFactory = GeometryFactory(PrecisionModel(), SRID.LONG_LAT)
+
+      val elevationByPlotId =
+          plots
+              .mapNotNull { plot ->
+                val swCorner = geometryFactory.createPoint(plot.boundary.coordinates[SOUTHWEST])
```

Comment:
Another option: `plot.boundary.centroid()` to get the point at the center of the plot.